ci_testing/run_cpu.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ==============================================================================
+
+set -e
+set -x
+
+N_JOBS=$(grep -c ^processor /proc/cpuinfo)
+
+echo ""
+echo "Bazel will use ${N_JOBS} concurrent job(s)."
+echo ""
+
+export CC_OPT_FLAGS='-mavx'
+export TF_NEED_CUDA=0 # TODO: Verify this be used in GPU custom-op?
+
+######### Python3 Tests
+# TODO: tf-nightly-2.0-preview is unavailable for python3.4
+
+
+#export PYTHON_BIN_PATH=`which python3`
+#/bin/bash configure.sh --py_version=3
```

Comment:
I think it will be easier if we set the python from the script that calls this one. That way, we have a few scripts here, and we can run the same script for py2/3. We can also install the tf preview from there-- but up to you; maybe it's better to pip install here, so that if you want to change TF versions (eg when it is no longer the preview), you can control that from here. That would also work presumably if we have separate GPU testing scripts, as you can install the GPU package in the same way.

Reply:
Because our BUILD depends on a `.bazelrc` file that is created by `configure.sh` it's probably best that it's called in the script. I also like the idea of controlling the TF version because of issues like https://github.com/tensorflow/addons/pull/37 where it's important that we pin to a specific version once they're available.